README.md

```diff
@@ -39,12 +39,27 @@ Node, or NPM. So we'll just assume you already have them all running well.
 
 ### Compiling
 
-Assuming you already have Node and NPM working, compilation is easy. Use the
-following shell code:
+First you must get a copy of the pomf code.  To do so, clone this git repo.
+If you have no desire for any submodules, which includes the PHP backend, simply run
```

Comment:
This should not be the first option in that case, perhaps doing git clone --recursive as the first command

then saying "If you have no desire for any submodules, which includes the PHP backend, simply then run:"